server.py

```diff
@@ -24,27 +27,29 @@ def do_GET(self):
             'data': 'Hello I am a swarm node.'
         }
 
-        self._set_headers()
+        self.send_response(200)
+        self.send_header('Access-Control-Allow-Origin', '*')
+        self.end_headers()
         self.wfile.write(json.dumps(response))
 
     def do_POST(self):
 
         request_path = self.path
 
         request_headers = self.headers
-        content_length = request_headers.get("Content-length")
```

Comment:
Why would you change the way to specify string literals? You have to minimize the functional changes. Once you wish to introduce syntax tweaks, you can always send another pull request(s).

Reply:
Got it. I'll remove the style tweak on string literals and send pull request again.